epochtalk/epochtalk
# Deletion

Category Deletion

Testing procedure:
* Log in as an admin
* Create a new Category
* Save
* Ensure the category exists after saving
* Delete Category
* Ensure category is deleted
* Create a category and delete it before saving.
* Ensure extra calls to the server were not made

To test the other commits:
* Go to a posts page
* Ensure that only one call to the thread is made
* Duplicate the test for threads page

* Go to a posts page as a non admin
* Ensure that a call to boards.all isn't made
* Repeat test as an admin user

* Go to a posts page with multiple pages
* Ensure that all calls to the server succeed
